app/assets/javascripts/utilities/localDateTime.js

```diff
@@ -25,18 +25,44 @@ function timestampToLocalDateTime(timestamp, locale, options) {
   }
 }
 
-function addLocalizedDateTimeToElementsTitles(elements, timestampAttribute) {
+function timestampToLocalDateTimeLong(timestamp) {
   // example: "Wednesday, April 3, 2019, 2:55:14 PM"
-  var timeOptions = {
+
+  return timestampToLocalDateTime(timestamp, navigator.language, {
     weekday: 'long',
     year: 'numeric',
     month: 'long',
     day: 'numeric',
     hour: 'numeric',
     minute: 'numeric',
     second: 'numeric',
-  };
+  });
+}
+
+function timestampToLocalDateTimeShort(timestamp) {
```

Comment:
I have added this util function which uses ```timestampToLocalDateTime``` to give shorter version of date similar to what you see in post (the helper used by post is rails helper). This could be used to replace duplicate date display logic in other places.